soccer_vision_msgs/msg/Obstacle.msg

```diff
@@ -0,0 +1,14 @@
+# Describes a detected obstacle from vision modules, in image coordinates.
```

Comment:
How do we define obstacles? Are laptops on the field obstacles, are robots also obstacles, is the referee or robot handler a special obstacle?

Reply:
I would define obstacles as anything the robot has to avoid when walking around the field, and wouldn't go into the specifics of what it is.

Reply:
I would prefer obstacles being anything that is not defined elsewhere (such as robots or goalposts).
That would also mean, that other things that may be specified in an own message in the future (e.g. `Human`) are no longer a `Obstacle`.

Reply:
I like @jaagut 's suggestion. So a node that does pathplanning should, rather than just listen to obstacles, listen to all robots, goalposts, and obstacles.